test/models/badge_test.rb

```diff
@@ -2,8 +2,10 @@
 
 class BadgeTest < ActiveSupport::TestCase
   test "percentage_awardees" do
-    badge = create :badge, num_awardees: 123_456
-    assert_equal 15.44, badge.percentage_awardees
+    create_list(:user, 21)
+
+    badge = create :badge, num_awardees: 4
+    assert_equal 9.76, badge.percentage_awardees
```

Comment:
Sorry, I don't generally read Ruby. What's this test saying? That 4 is 9.76% of something? Is that something 41?